builder/azure/arm/config.go

```diff
@@ -1291,6 +1309,18 @@ func assertRequiredParametersSet(c *Config, errs *packersdk.MultiError) {
 		}
 	}
 
+	if c.SecurityType != "" {
+		if ok, err := assertAllowedSecurityType(c.SecurityType, "security_type"); !ok {
+			errs = packersdk.MultiErrorAppend(errs, err)
+		}
+	}
+
+	if c.SecurityEncryptionType != "" {
+		if ok, err := assertAllowedSecurityEncryptionType(c.SecurityEncryptionType, "security_encryption_type"); !ok {
```

Comment:
I like how you've broken out this and `assertAllowedSecurityType` into their own functions, but I think the second parameter in both of these functions might not be needed, since they are only called once with those parameters, I think you can just hardcode that field name in the string

Reply:
Yep, great point. I will remove the param in my next commit.

Reply:
Okay, those parameters were removed, and the setting value was instead hard-coded in the error string — changes made in [f26bf29](https://github.com/hashicorp/packer-plugin-azure/pull/373/commits/f26bf290ba52b1bb16aa438428f9ab26edea3324).